sqlite3_flutter_libs/linux/CMakeLists.txt

```diff
@@ -13,9 +13,17 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(sqlite3)
 
+FetchContent_Declare(
+  spellfix1
+  URL https://raw.githubusercontent.com/sqlite/sqlite/branch-3.39/ext/misc/spellfix.c
+  DOWNLOAD_NO_EXTRACT TRUE
+)
+FetchContent_MakeAvailable(spellfix1)
```

Comment:
I think we need the same logic for Windows in this repository, right?

Android should be done in the `sqlite-native-libraries` repository. I include that from maven central here because we're not allowed to ship binaries to pub.dev and since users can't be trusted to get their NDK install right (so we can't ship sqlite3 sources).